llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -69,6 +69,7 @@ class LoopVectorizationCostModel;
 class LoopVersioning;
 
 struct VPCostContext;
+struct HistogramInfo;
```

Comment:
might have missed it, but not used in latest version in the file?

Reply:
done